checklist.md

```diff
@@ -0,0 +1,118 @@
+# Сheck Your Code Against the Following Points
+
+## Don't Push db files
+
+Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension).
+Don't forget to add `.gitignore` file before pushing. 
+
+## Don't Forget to Add Migrations to your PR
+This is a required for the tests to pass.
+
+## Code Style
+1. Implement `Meta` class only after you defined all attributes.
+
+Good example:
+```python
+class Order(models.Model):
+    created_at = ...
+    user = ...
+
+    class Meta:
+        pass
+```
+
+Bad example:
+```python
+class Order(models.Model):
+    created_at = ...
+
+    class Meta:
+        pass    
+
+    user = ...
+```
+
+2. Use `AUTH_USER_MODEL` instead of `User`, it is the best practice.
+
+Good example:
+```python
+class Order(models.Model):
+    customer = models.ManyToManyField(AUTH_USER_MODEL)
```

Comment:
get_user_model()?

Reply:
No, as I understood Danya told them to use `settings.AUTH_USER_MODEL)`